spec/spec_helper.rb

```diff
@@ -50,6 +51,8 @@
 
   config.include FactoryBot::Syntax::Methods
 
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
this beats installing the timecop to make predictable assertions about time in tests